### PR TITLE
Modified Quad Blaster Turret

### PR DESCRIPTION
**Pull request by Pointedstick**
Here's a new weapon for sale on Pirate worlds (And Trinket, I suppose): the Modified Quad Blaster turret.

modified quad blaster turret

Though its energy use is high and heat output is ridiculous, the absolute damage and DPS/ton are quite respectable for being a Tier 1 human turret. Of course, you're going to need to install a lot of cooling...

I stuck a few on some Marauder ships, which seemed appropriate.

Turn rate is "post-https://github.com/endless-sky/endless-sky/pull/3271" and should not require modification when @endless-sky reduces turret turn rates across the board.

Bolt speeds are not reflective of https://github.com/endless-sky/endless-sky/pull/3307--just in case @endless-sky opts not to merge it.

HiDPI assets/pull request: https://github.com/endless-sky/endless-sky-high-dpi/pull/32

Blender and GIMP assets: [Modified Quad Blaster Turret assets.zip](https://github.com/endless-sky/endless-sky/files/1519704/Modified.Quad.Blaster.Turret.assets.zip)

### CODE REVIEW

File: data/weapons.txt [hunk 1]
```diff
@@ -139,6 +139,32 @@ outfit "Modified Blaster Turret"
 		"hull damage" 8
 	description "This is a turreted version of the Modified Blaster, which is an Energy Blaster modified for greater power at the cost of higher energy consumption and heat. These turrets are popular with pirates and others who are trying to cram as much firepower into their ships as possible."
 
+outfit "Quad Mod Blaster Turret"
```
Comment (tehhowch) [inline data/weapons.txt]: `Mod` -> `Mod.` IMO

**Comment by Nescio0** [comment]
Is there a maximum character limit for names? If not, I'd prefer "Modified" instead of "Mod".

**Comment by Pointedstick** [comment]
There actually is room for the full name, so I've changed it to use that. :+1:

**Comment by KiLEdEnNis** [comment]
I personally prefer "Modified Quad Blaster Turret", anyone else?

**Comment by Pointedstick** [comment]
Yeah, that _is better_. Changing now.

**Comment by Bladewood** [comment]
The image should be named `quad mod blaster turret` (or `mod quad blaster turret`, depending on your preferences), in keeping with the modified blaster and turret.

**Comment by Pointedstick** [comment]
The images have already been renamed.

**Comment by Bladewood** [comment]
Doesn't look like it.
notmodturret

**Comment by Pointedstick** [comment]
Oh I see what you mean now. Renaming the files yet again...

**Comment by Amazinite** [comment]
*Looks at firing heat/sec*
Oh lord.

**Comment by Pointedstick** [comment]
Check out the DPS per ton though! I like how extreme a weapon it is.

**Comment by Amazinite** [comment]
Certainly an interesting weapon, to say the least. As far as blasters go, I like it. They're actually slightly *more* efficient in terms of heat and energy usage / DPS (and I would expect them to be slightly less so), but I'd let it slide given how insane it is already.

**Comment by Amazinite** [comment]
Just a thought: I've had the idea in the past of creating outfits that can only be found on Marauder ships. Perhaps you could make additional 2 or 3 Marauder variants that use this turret and slip them in to the fleets for people to look for, then not sell the turret on pirate worlds. :)

**Comment by Pointedstick** [comment]
I would be happy to create even more insane Marauder versions of things (Octo Modified Blaster Turret! Each one needs a liquid Nitrogen cooler!), but my plan right now is to create many outfits that can only be found on Pirate worlds, to make them more interesting and reward players who risk visiting them--or who decide to become a pirate!

**Comment by Amazinite** [comment]
How about we finish up that Quad Javelin Turret? #2535

**Comment by solardawning** [comment]
I like the idea of more Pirate world content.
I may make a few sets of 'Modified' ship stats (in the vein of the Modified Argosy, having been converted from a freighter to an improvised warship), to represent pirates making due with whatever they can get their hands on.

**Comment by KiLEdEnNis** [comment]
keep in mind: such stats should have modified ship models alongside them, and requires adding said ship and possible variants to fleets and maybe a shipyard

**Comment by solardawning** [comment]
Yeah, I have no experience making the models, which to date is what has stayed my hand. The rest is simple, though. Some of the 'old' Betelgeuse ships in use for centuries would be good candidates to have been heavily modified by pirates: I could see the Star Queen made into a pirate captain's flagship (with most of the luxuries stripped out, except for the captain's cabin. ;) )

**Comment by endless-sky** [comment]
I'm really not sold on this one. Mod blasters are way too inefficient to be much use anyway except on small ships where there's heat dissipation to spare. (And, the name is just too long.)

**Comment by Pointedstick** [comment]
I dunno, I think the improved damage and range actually make it pretty decent, if you can stick a big cooler on your ship--or already have one. Besides, we need more Pirate outfits anyway.

**Comment by Lineth** [comment]
Reposting because I am dumb:
 I did maker a personal mod of the mod quad blaster turret with stats based on the current mod blaster line and it seems actually competitive with the Hai Pulse turret. This looks like it's a poor man's Plasma Turret, which IMO is not bad. Maybe a another smuggler's bay/custom pirate candidate?

I'd also like a scaled-up version of the blaster, perhaps. Maybe two of them bolted together, or even three to act as what amounts to a small space shotgun, or a funsize proton cannon.

**Comment by Pointedstick** [comment]
> It looks like you missed adding a crew requirement to  the turret, as well. All other offensive turrets have a crew  requirement of 1, except Dual Sunbeam, which takes 2 crew.

Looks like I remembered it for this one.

**Comment by tehhowch** [comment]
Oof your PR got a bit nuked, you may want to reset to an earlier commit. How are you going about conflict resolution?
I generally do all my conflict resolution locally (`git fetch upstream, git merge upstream/master, <edits> & git add ...., git commit -m 'Merge upstream/master', git push origin`) rather than do anything with the web interface.

**Comment by Pointedstick** [comment]
NooOoOooooo git [shakes fist]

**Comment by Pointedstick** [comment]
Thanks for your help, @tehhowch.

**Comment by Nescio0** [comment]
Separately, could you design a turret with eight ordinary blasters? Statistically basically double the ordinary quad blaster turret, it would be able to inflict an amount of damage somewhere between heavy laser and thrasher turrets, which could make it occassionally an interesting choice, although still highly inefficient.

**Comment by Pointedstick** [comment]
Sure, go ahead. :slightly_smiling_face: Though given @endless-sky's comments, so far I'm not optimistic about even this making it in...

**Comment by KiLEdEnNis** [comment]
please stop at 8, don't do 16 or 32 Blasters on a turret

**Comment by endless-sky** [comment]
I really don't feel like the human weapons need even more blaster variants in addition to the 5 existing ones. Also, the blaster buff and the addition of the proton turret make other good projectile options available.

**Pull request closed without merging**